README.md

```diff
@@ -182,6 +182,16 @@ If you find PAWLS helpful for your research, please consider cite PAWLS.
 }
 ```
 
+### Troubleshooting
+
+1. Windows EOL format (CRLF) vs Linux (LF)
+
+Sometimes you may wondering why building docker image is not working and `ui` part is taking down. It's because some files were saved in incorrect format. To healing such files just go to the repo's root and paste following in terminal.
```

Comment:
```suggestion
The application was developed for Linux, and might fail to start on Windows because of line-ending differences.

To fix this, run this command from the root of the repository:
```